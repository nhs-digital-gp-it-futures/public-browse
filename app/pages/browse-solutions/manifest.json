{
  "title": "GP IT Futures framework",
  "description": "There are two types of Catalogue Solution on the GP IT Futures framework. You can choose to view all those available or only the ones which have all 6 Foundation Capabilities - the minimum requirement to enable a GP practice to operate.",
  "foundationPromoHeading": "View Foundation Solutions",
  "foundationPromoDescription": "Find Catalogue Solutions that meet all Foundation Capabilities",
  "allPromoHeading": "View all Catalogue Solutions",
  "allPromoDescription": "Find Catalogue Solutions with the Capabilities to meet your needs",
  "comparePromoHeading": "Compare all Catalogue Solutions",
  "comparePromoDescription": "Get an overview of all the Catalogue Solutions available and filter them by the Capabilities they offer"
}
