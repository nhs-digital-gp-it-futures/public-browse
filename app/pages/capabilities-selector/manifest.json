{
  "title": "Catalogue Solutions on the GP IT Futures framework â€“ Capability selector",
  "description": "Select the Capabilities that match the needs of your organisation. Any Catalogue Solutions found will have proven they can address these needs and, in some cases, will do even more."
}
