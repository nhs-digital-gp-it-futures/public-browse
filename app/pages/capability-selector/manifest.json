{
  "title": "Catalogue Solutions â€“ Capability selector",
  "description": "Select the Capabilities that match the needs of your organisation. The Catalogue Solutions found will have proven they can address these needs and, in some cases, will do even more."
}
