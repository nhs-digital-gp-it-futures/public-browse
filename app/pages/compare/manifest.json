{
  "title": "Compare all Catalogue Solutions",
  "description": "Use our compare document to help find what youâ€™re looking for. It gives an overview of all the Catalogue Solutions available and lets you filter them by the Capabilities they offer.",
  "compareButtonText": "Compare Solutions (.xlsx)",
  "backLinkText": "Back"
}
