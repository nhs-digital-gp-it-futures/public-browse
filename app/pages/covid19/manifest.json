{
  "title": "Catalogue Solutions to help with coronavirus",
  "description": "In response to the coronavirus emergency, the Catalogue Solutions listed below have been through accelerated assurance from NHS Digital, which covers key characteristics such as Information Governance and Clinical Safety.",
  "insetText": [
    "Full assurance is in progress and we’ll advise when this is complete, however you should feel confident in seeking to deploy these Catalogue Solutions immediately.",
    "We’ve carried out an assessment of the Catalogue Solutions or their Additional Services to ensure they meet either an NHS defined eConsultation Capability or aProductivity Capability defined by the supplier.",
    "The Solutions are under constant development, so we’ll continue to assess their functionality."
  ],
  "buttonText": "Compare Solutions (PDF)",
  "buttonHref": "/$web/content/covid19-compare-solutions.pdf",
  "subtext": "Use the compare Solutions button to see an overview of the key features offered by each of these Catalogue Solutions."
}
