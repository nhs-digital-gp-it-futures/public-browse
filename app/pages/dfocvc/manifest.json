{
  "title": "DFOCVC framework - results",
  "description": "These are the online and video consultation Catalogue Solutions available on the DFOCVC framework."
}
