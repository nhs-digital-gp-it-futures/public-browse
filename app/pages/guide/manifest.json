{
  "title": "Buyer’s Guide",
  "description": "Read the following information before using the Buying Catalogue. It’ll help you find the information you need to make an informed decision when procuring.",
  "advice": [
    "All products and services listed on the Buying Catalogue are known as Catalogue Solutions. Each Solution has a certain number of Capabilities which can address business needs in primary care.",
    "To prove these Capabilities, Suppliers must demonstrate that their Catalogue Solution can satisfy Epics which detail the functions required to successfully achieve a Capability.",
    "There is a link to further Buyer Guidance in the 'Learn more' section of this page."
  ],
  "subtext": "These are the key terms you need to understand to use the Buying Catalogue effectively:",
  "sections": [
    {
      "id": "catalogue-solution",
      "title": "Catalogue Solution",
      "subsections": [
        {
          "id": "description",
          "description": [
            "A Catalogue Solution is a product or service that has successfully demonstrated compliance with Capabilities and Standards that apply to it. The Catalogue Solution will only be made available on the Buying Catalogue once it has demonstrated its compliance."
          ]
        },
        {
          "id": "associated-services",
          "title": "Associated Services",
          "description": [
            "Associated Services help you to implement or optimise the live operation of a Catalogue Solution and/or Additional Services, for example, training."
          ]
        },
        {
          "id": "additional-services",
          "title": "Additional Services",
          "description": [
            "Additional Services are add-ons that provide additional Capabilities to a Catalogue Solution for an extra cost.",
            "When you view Catalogue Solutions on the Buying Catalogue, if there are Additional Services available, information about their Capabilities and price will be presented to you.",
            "Additional Services may also have Associated Services."
          ]
        }
      ]
    },
    {
      "id": "capabilities",
      "title": "Capabilities",
      "subsections": [
        {
          "id": "capabilities-description",
          "description": [
            "Capabilities are how you can match a Catalogue Solution to a business need.",
            "For example, you may want to enable GP staff to make a digital record of any consultation with a patient. You can therefore identify Catalogue Solutions on the Buying Catalogue which offer the 'Recording Consultations' Capability.",
            "There are a number of pre-defined Capabilities, but there is also the Productivity Capability. This is defined by the Supplier when their Catalogue Solution does not map to a pre-defined Capability."
          ]
        },
        {
          "id": "learn-more",
          "description": [
            {
              "startText": "",
              "linkText": "Learn more about the Capabilities and Standards models for frameworks on the Buying Catalogue",
              "href": "https://gpitbjss.atlassian.net/wiki/spaces/DCSH/overview",
              "endText": ""
            }
          ]
        },
        {
          "id": "epics",
          "title": "Epics",
          "description": [
            "Epics precisely describe the functions of a Capability.",
            "For example, under ‘Recording Consultations’ some of the Epics that need to be met are ‘record Consultation information’ and ‘manage Consultation form templates’.",
            "There are two types of Epic: Must Epics which are required and May Epics which are optional."
          ]
        },
        {
          "id": "foundation-solution-set",
          "title": "Foundation Solution Set",
          "description": [
            "6 Foundation Capabilities are the minimum requirement to enable a GP practice to operate. All 6 Capabilities must be fulfilled to achieve Foundation Solution Set status."
          ]
        },
        {
          "id": "they-are",
          "description": [
            "They are:"
          ]
        },
        {
          "id": "bullet-list",
          "bulletlist": [
            "Appointments Management – GP",
            "Referral Management",
            "Resource Management",
            "Patient Information Maintenance",
            "Prescribing",
            "Recording Consultations"
          ]
        },
        {
          "id": "view-only-catalogue-solutions",
          "description": [
            "You can use the Buying Catalogue to only view Catalogue Solutions that offer the Foundation Solution Set (which meet the 6 Foundation Capabilities), or to view all the Catalogue Solutions available."
          ]
        }
      ]
    },
    {
      "id": "standards",
      "title": "Standards",
      "subsections": [
        {
          "id": "standards-describe",
          "description": [
            "Standards describe the technical or operating levels a Catalogue Solution must meet. Each Catalogue Solution must meet all Standards relevant to the Capabilities it offers.",
            "Therefore, if a Catalogue Solution is listed on the Buying Catalogue, a Buyer can be certain the standards have been met."
          ]
        }
      ]
    },
    {
      "id": "work-off-plans",
      "title": "Work-off Plans",
      "subsections": [
        {
          "id": "work-off-plans-describe",
          "description": [
            "A Catalogue Solution close to meeting a required Capability or compliance with a Standard can be given a Work-off Plan. This means the Supplier has agreed to complete any outstanding work in an agreed timeframe.",
            "Any such Catalogue Solution on the Buying Catalogue will have the details of its Work-off Plan included on its listing page. Buyers can take this in to consideration when browsing the Buying Catalogue."
          ]
        }
      ]
    },
    {
      "id": "integrations",
      "title": "Integrations",
      "subsections": [
        {
          "id": "nhs-assured-integrations",
          "title": "NHS Assured Integrations",
          "description": [
            "NHS Assured Integrations are the interoperability integrations specified and assured by the NHS: GP Connect, which allows GP practices to share and view clinical information, and IM1, which helps Catalogue Solutions integrate with other clinical systems."
          ]
        },
        {
          "id": "supplier-assured-integrations",
          "title": "Supplier Assured Integrations",
          "description": [
            "Supplier Assured Integrations are interoperability interfaces prepared by a Supplier and are not specified or assured by the NHS."
          ]
        }
      ]
    },
    {
      "id": "learn-more",
      "title": "Learn More",
      "subsections": [
        {
          "id": "learn-more",
          "description": [
            "You can refer to this guidance at any time using the Buyer’s Guide link in the footer of every page.",
            {
              "startText": "You can access",
              "linkText": "further Buyer Guidance",
              "href": "https://gpitbjss.atlassian.net/wiki/spaces/BG/overview",
              "endText": " on Capabilities and Standards, contractual requirements, funding and procurement methods."
            }
          ]
        }
      ]
    },
    {
      "id": "contact-us",
      "title": "Contact us",
      "subsections": [
        {
          "id": "contact-us-paragraph",
          "description": [
            {
              "startText": "If you experience any technical issues when using this website, please contact NHS Digital’s Exeter Helpdesk at",
              "linkText": "exeter.helpdesk@nhs.net",
              "href": "mailto:exeter.helpdesk@nhs.net",
              "endText": "or call 0300 303 4034."
            }
          ]
        }
      ]
    }
  ]
}
