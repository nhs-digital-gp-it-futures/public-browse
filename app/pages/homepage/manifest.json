{
  "heroHeading": "Buying Catalogue",
  "heroText": "Find out what clinical information solutions are available from procurement frameworks on the Buying Catalogue.",
  "title": "About the Buying Catalogue",
  "description": [
    "The NHS Digital Buying Catalogue is an online marketplace where you can purchase clinical information systems that meet the digital needs of primary care.",
    "All the listed products – known as Catalogue Solutions – have been assessed against a set of Capabilities defined by the NHS and a set of Standards defined by NHS Digital.",
    "The functionality of the Buying Catalogue will be enhanced and the number of Catalogue Solutions available will increase over time."
  ],
  "adminPromoHeading": "Manage organisations and users",
  "adminPromoDescription": "Add new organisations and user accounts or edit existing details",
  "guidePromoHeading": "Buyer’s Guide",
  "guidePromoDescription": "Terminology you’ll need to know when procuring Catalogue Solutions",
  "orderFormPromoHeading": "Create or manage orders",
  "orderFormPromoDescription": "Place a new order or complete an order you've already started (log in required)",
  "vaccinationsPromoHeading": "Coronavirus vaccinations",
  "vaccinationsPromoDescription": "Find Catalogue Solutions that help organise coronavirus vaccinations",
  "viewSolutionsPromoHeading": "GP IT Futures framework",
  "viewSolutionsPromoDescription": "Find and compare Catalogue solutions available on the GP IT Futures framework",
  "dfocvcHeading": "DFOCVC framework",
  "dfocvcDescription": "Find online and video consultation Catalogue Solutions on the DFOCVC framework"
}
