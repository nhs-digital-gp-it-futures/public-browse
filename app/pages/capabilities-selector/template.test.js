import { createTestHarness } from '../../test-utils/testHarness';
import manifest from './manifest.json';

const setup = {
  template: {
    path: 'pages/capabilities-selector/template.njk',
  },
};

describe('capabilities-selector page', () => {
  it('should render a backLink to the solutions page', createTestHarness(setup, (harness) => {
    const context = {};

    harness.request(context, ($) => {
      const backLink = $('[data-test-id="go-back-link"]');
      expect(backLink.length).toEqual(1);
      expect(backLink.text().trim()).toEqual('Go back to previous page');
      expect(backLink.find('a').attr('href')).toEqual('/solutions');
    });
  }));

  it('should render the title', createTestHarness(setup, (harness) => {
    const context = { ...manifest };

    harness.request(context, ($) => {
      const title = $('[data-test-id="capabilities-selector-page-title"]');
      expect(title.length).toEqual(1);
      expect(title.text().trim()).toEqual(manifest.title);
    });
  }));

  it('should render the description', createTestHarness(setup, (harness) => {
    const context = { ...manifest };

    harness.request(context, ($) => {
      const description = $('[data-test-id="capabilities-selector-page-description"]');
      expect(description.length).toEqual(1);
      expect(description.text().trim()).toEqual(manifest.description);
    });
  }));

  it('should render the capabilities selector', createTestHarness(setup, (harness) => {
    const context = {
      ...manifest,
      column1: [{
        reference: 'C5',
        version: '1.0.1',
        name: 'Appointments Management - GP',
        isFoundation: true,
      }],
    };

    harness.request(context, ($) => {
      const capabilitiesSelectorComponent = $('[data-test-id="capabilities-selector"]');
      expect(capabilitiesSelectorComponent.length).toEqual(1);
    });
  }));

  it('should render the button', createTestHarness(setup, (harness) => {
    const context = { ...manifest };

    harness.request(context, ($) => {
      const capabilitiesSelectorButton = $('[data-test-id="capabilities-selector-continue-button"] a');
      expect(capabilitiesSelectorButton.length).toEqual(1);
      expect(capabilitiesSelectorButton.text().trim()).toEqual('Continue');
    });
  }));
});
