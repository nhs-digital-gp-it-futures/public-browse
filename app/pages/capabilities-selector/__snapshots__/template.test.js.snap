// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`capabilities-selector page should render the content 1`] = `
"<main class=\\"nhsuk-main-wrapper\\" id=\\"maincontent\\" data-test-id=\\"main-content\\">

    
  <div class=\\"nhsuk-width-container\\">
    <div class=\\"nhsuk-u-padding-top-5\\" data-test-id=\\"go-back-link\\">
      <div class=\\"nhsuk-back-link\\">
  <a class=\\"nhsuk-back-link__link\\" href=\\"./\\">
  <svg class=\\"nhsuk-icon nhsuk-icon__chevron-left\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" height=\\"24\\" width=\\"24\\">
    <path d=\\"M8.5 12c0-.3.1-.5.3-.7l5-5c.4-.4 1-.4 1.4 0s.4 1 0 1.4L10.9 12l4.3 4.3c.4.4.4 1 0 1.4s-1 .4-1.4 0l-5-5c-.2-.2-.3-.4-.3-.7z\\"></path>
  </svg>
  Go back</a>
</div>

    </div>
    <h1 class=\\"nhsuk-u-margin-top-6 nhsuk-u-margin-bottom-4\\" data-test-id=\\"capabilities-selector-page-title\\">Catalogue Solutions on the GP IT Futures framework â€“ Capability selector</h1>
    <form method=\\"post\\">
      <input type=\\"hidden\\" name=\\"_csrf\\" value=\\"\\">
      <div class=\\"nhsuk-grid-row\\">
        <fieldset class=\\"nhsuk-fieldset\\" aria-describedby=\\"capabilities-hint\\" data-test-id=\\"capabilities-selector-fieldset\\">
          <legend class=\\"nhsuk-heading-l nhsuk-u-font-size-22 nhsuk-grid-column-two-thirds\\" data-test-id=\\"capabilities-selector-page-description\\">
            Select the Capabilities that match the needs of your organisation. Any Catalogue Solutions found will have proven they can address these needs and, in some cases, will do even more.
          </legend>
          <div class=\\"nhsuk-hint nhsuk-grid-column-two-thirds\\" id=\\"capabilities-hint\\">
            Select all options that are relevant to you.
          </div>
          
  <div id=\\"capabilities-selector\\" data-test-id=\\"capabilities-selector\\">
    <div data-test-id=\\"capabilities-checkbox-column-1\\" class=\\"nhsuk-grid-column-one-half\\">
      





  

  
<div class=\\"nhsuk-form-group\\">

  <div class=\\"nhsuk-checkboxes\\">
    
    
    
    
    
    
    <div class=\\"nhsuk-checkboxes__item\\">
      <input class=\\"nhsuk-checkboxes__input\\" id=\\"column-1-1\\" name=\\"capabilities\\" type=\\"checkbox\\" value=\\"C1\\">
      <label class=\\"nhsuk-label nhsuk-checkboxes__label\\" for=\\"column-1-1\\">
        Capability 1
      </label>
    </div>
    
    
    
    
    
    
    
    
    <div class=\\"nhsuk-checkboxes__item\\">
      <input class=\\"nhsuk-checkboxes__input\\" id=\\"column-1-2\\" name=\\"capabilities\\" type=\\"checkbox\\" value=\\"C2\\">
      <label class=\\"nhsuk-label nhsuk-checkboxes__label\\" for=\\"column-1-2\\">
        Capability 2
      </label>
    </div>
    
    
    
  </div>

</div>

    </div>
    <div data-test-id=\\"capabilities-checkbox-column-2\\" class=\\"nhsuk-grid-column-one-half\\">
      





  
<div class=\\"nhsuk-form-group\\">

  <div class=\\"nhsuk-checkboxes\\">
    
    
    
    
    
    
    <div class=\\"nhsuk-checkboxes__item\\">
      <input class=\\"nhsuk-checkboxes__input\\" id=\\"column-2-1\\" name=\\"capabilities\\" type=\\"checkbox\\" value=\\"C3\\">
      <label class=\\"nhsuk-label nhsuk-checkboxes__label\\" for=\\"column-2-1\\">
        Capability 3
      </label>
    </div>
    
    
    
  </div>

</div>

    </div>
  </div>

        </fieldset>
      </div>
      <div data-test-id=\\"capabilities-selector-continue-button\\">
        


  
    
  




<button class=\\"nhsuk-button nhsuk-button--secondary\\" type=\\"submit\\">
  Continue
</button>



        </div>
      </form>
    </div>


  </main>"
`;
